src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -135,6 +180,7 @@ public void mercury_search_focusOnTryAnotherSearchWhenNoResults() {
     Assertion.assertTrue(searchResults.getSearch().getSearchPhrase().isEmpty());
   }
 
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
   @Test(groups = "mercury_search_redirectToNewResultsPageFromNoResults")
   public void mercury_search_redirectToNewResultsPageFromNoResults() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)